dbt-cta/facebook_pages/models/1_cta_full_refresh/page_base.sql

```diff
@@ -0,0 +1,229 @@
+{% set partitions_to_replace = [
+    "timestamp_trunc(current_timestamp, day)",
+    "timestamp_trunc(timestamp_sub(current_timestamp, interval 1 day), day)"
+] %}
+{{ config(
+    cluster_by = "_airbyte_extracted_at",
+    partition_by = {"field": "_airbyte_extracted_at", "data_type": "timestamp", "granularity": "day"},
+    partitions = partitions_to_replace,
+    unique_key = "_airbyte_page_hashid"
+) }}
+
+-- Final base SQL model
+-- depends_on: {{ ref('page_ab1') }}
+select
+    `_airbyte_page_hashid`,
```

Comment:
is there a reason the column names are wrapped in these ticks `?

Reply:
oh yeah, at one point I was hitting an error because the name of a field was a reserved name in BQ (or whatever it's called) - like "group" or something? - and so the SQL wasn't being parsed correctly without backticks to indicate field names. So it seemed easiest to just add the backticks everywhere 🫠